### ISSUE TITLE
New useful information

### ISSUE THREAD

**Issue by ale-lit** [Schweinepriester/github-profile-achievements#15]
1. If all the conditions are met correctly, the reward is issued instantly.
2. To obtain a Galaxy Brain, the following conditions must be met:
    - Only answers in other people's questions selected by other participants are taken into account
    - If there are already more than 2 of your answers to your questions, then you will not be able to receive the reward (until you cancel them)
    - The brain is issued only on the condition that it will be exactly the answer No. 2, and it will meet the requirements of paragraph No. 1

**Comment by ale-lit** [comment]
sorry for my bad english

**Comment by Schweinepriester** [comment]
no worries, thank you for the information!

it also looks like you were able to score YOLO via https://github.com/ale-lit/book-shop-laravel/pull/4 :)
which defeats my theory that it only can be achieved in other peoples repos.

I was searching if you got twitter (find me [here](https://twitter.com/schweinepriestr) if you have) or another platform where we can instant message, wanted to ask if I may add you to https://github.com/Schweinepriester/achievement-testarea so I can pretend to request a review from you? :)

**Comment by ale-lit** [comment]
> it also looks like you were able to score YOLO via https://github.com/ale-lit/book-shop-laravel/pull/4 :)
> which defeats my theory that it only can be achieved in other peoples repos.
Yes ) To complete this task, I had to add a third-party reviewer for the PR (otherwise the task did not count)

**Comment by ale-lit** [comment]
Twitter is not work now in my country ( And so, of course, I'm ready)

**Comment by Schweinepriester** [comment]
Alright, https://github.com/Schweinepriester/achievement-testarea/pull/4 worked, thank you very much!
Pretty minimal case as well!

Will answer https://github.com/Schweinepriester/github-profile-achievements/discussions/6 now.
I think I'm gonna do a "specific instructions" section in the main README.

**Comment by ale-lit** [comment]
Good! Congratulations 🎉 ))

**Comment by ayushgupta9198** [comment]
@Schweinepriester Please let us know how we can earn this YOLO badge in our profile , i have tried for pull request but wont work ....
Thanks in advance

**Comment by Schweinepriester** [comment]
@ayushgupta9198 https://github.com/Schweinepriester/github-profile-achievements/discussions/6#discussioncomment-2934257 is still the way to the best of our knowledge.

**Comment by warriorwizard** [comment]
Here is how i earned Galaxy Brain:

***repeat the process for two separate discussions***

1. used another account to start a discussion in the repository
2. Mark the repository as Q&A
3. Answer the discussion from the account (in which u want Galaxy Brain)
4. Go back to ur other account and mark the above mentioned reply as answer

wolaa... u hava ur Galaxy Brain

**Comment by Aslin-R** [comment]
To unlock the Galaxy Brain badge on GitHub, you need to answer at least 2 discussions. The discussions must be in a public repository, and you cannot mark your own answers to your own questions.
